tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -26,26 +26,70 @@
 
 namespace Tests\Integration\Behaviour\Features\Context\Domain;
 
+use AdminController;
+use Behat\Gherkin\Node\TableNode;
 use Context;
+use FrontController;
 use Order;
 use OrderState;
+use PrestaShop\PrestaShop\Core\Domain\Cart\Exception\CartConstraintException;
+use PrestaShop\PrestaShop\Core\Domain\Employee\Exception\InvalidEmployeeIdException;
 use PrestaShop\PrestaShop\Core\Domain\Order\Command\AddOrderFromBackOfficeCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Command\BulkChangeOrderStatusCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Command\UpdateOrderStatusCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Exception\OrderException;
+use PrestaShop\PrestaShop\Core\Domain\Order\Invoice\Command\GenerateInvoiceCommand;
 use PrestaShop\PrestaShop\Core\Domain\Order\Product\Command\AddProductToOrderCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Query\GetOrderForViewing;
+use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderForViewing;
 use PrestaShop\PrestaShop\Core\Domain\Order\ValueObject\OrderId;
-use Tests\Integration\Behaviour\Features\Context\SharedStorage;
-use PrestaShop\PrestaShop\Core\Domain\Order\Invoice\Command\GenerateInvoiceCommand;
+use PrestaShopDatabaseException;
+use PrestaShopException;
 use Product;
+use RuntimeException;
+use stdClass;
+use Tests\Integration\Behaviour\Features\Context\SharedStorage;
 
 class OrderFeatureContext extends AbstractDomainFeatureContext
 {
+    private const ORDER_STATUS_MAP = [
+        1 => 'Awaiting bank wire payment',
+        5 => 'Delivered',
+    ];
+
+    /**
+     * @BeforeScenario
+     */
+    public function before()
+    {
+        // needed because if no controller defined then CONTEXT_ALL is selected and exception is thrown
+        /** @var AdminController|FrontController $adminControllerTestDouble */
+        $adminControllerTestDouble = new stdClass();
+        $adminControllerTestDouble->controller_type = 'admin';
+        $adminControllerTestDouble->php_self = 'dummyTestDouble';
+        Context::getContext()->controller = $adminControllerTestDouble;
```

Comment:
Waho 😭it's heart-breaking we have to use this double
But I guess we cant do it without

Nice finding though 👍